File: Python/1040.Average3.py
nota1, nota2, nota3, nota4 = map(float, input().split())

media = (nota1*2 + nota2*3 + nota3*4 + nota4*1)/10

print("Media: {:.1f}".format(media))

if(media >= 7.0):
    print("Aluno aprovado.")
elif(media < 5.0):
    print("Aluno reprovado.")
elif(media < 7.0 and media >= 5.0):
    print("Aluno em exame.")
    notaRec = float(input())
    print("Nota do exame: {:.1f}".format(notaRec))
    mediaFinal = (media+notaRec)/2
    if(mediaFinal >= 5.0):
        print("Aluno aprovado.")
        print("Media final: {:.1f}".format(mediaFinal))
    else:
        print("Aluno reprovado.")
        print("Media final: {:.1f}".format(mediaFinal))

File: C/1040.Average3.c
#include <stdio.h>

int main(void){
    double nota1, nota2, nota3, nota4, notaRec, media, mediaFinal;

    scanf("%lf %lf %lf %lf", &nota1, &nota2, &nota3, &nota4);

    media = (nota1*2 + nota2*3 + nota3*4 + nota4*1)/10;

    printf("Media: %.1f\n", media);

    if(media >= 7.0){
        printf("Aluno aprovado.\n");
    }else if(media < 5.0){
        printf("Aluno reprovado.\n");
    }else if(media >= 5.0 && media < 7.0){
        printf("Aluno em exame.\n");
        scanf("%lf", &notaRec);
        mediaFinal = (media+notaRec)/2;
        printf("Nota do exame: %.1lf\n", notaRec);
        if(mediaFinal > 5.0){
            printf("Aluno aprovado.\n");
            printf("Media final: %.1lf\n", mediaFinal);
        }else{
            printf("Aluno reprovado.\n");
            printf("Media final: %.1lf\n", mediaFinal);
        }
    }

    return 0;
}